Examples/Calc/CalcNative/Service.h

```diff
@@ -0,0 +1,80 @@
+#include "Schema.h"
+#include <IPC/SharedMemory.h>
+#include <memory>
+#include <exception>
+#include <sstream>
+#include <iostream>
+
+
+namespace Calc
+{
+    class Service
+    {
+    public:
+        explicit Service(std::shared_ptr<IPC::SharedMemory> memory)
+            : m_memory{ std::move(memory) }
+        {
+            std::cout << "Service constructed" << std::endl;
+        }
+
+        Service(const Service& other) = delete;
+        Service& operator=(const Service& other) = delete;
+
+        Service(Service&& other) = default;
+        Service& operator=(Service&& other) = default;
+
+        ~Service()
+        {
+            if (m_memory)
+            {
+                std::cout << "Service destructed" << std::endl;
+            }
+        }
+
+        template <typename Callback>
+        void operator()(const Request& request, Callback&& callback) const
+        {
+            Response response;
+            std::stringstream text;
```

Comment:
minor: std::string?

Reply:
Need some basic formatting (don't want to use `std::to_string` here).